### ISSUE TITLE
Migrate from bower to npm

### ISSUE THREAD

**Issue by asantos00** [binhums/ember-cli-mapbox#54]
Hi! Great job with this addon :)

I was wondering if you guys are thinking about migrating bower packages to npm (as all the packages exist in npm and bower is getting deprecated). If it makes sense, I can help with that.

Hope to hear something from you

**Comment by planetexpress69** [comment]
@asantos00 Did you already fork and moved to npm? Just curious as I just wanted to switch to 2.18...

**Comment by asantos00** [comment]
Nope, not yet, we had some problems and it got stuck. I might have to get back at it to completely remove bower :/
